le_test.go

```diff
@@ -20,6 +21,21 @@ func TestConnectOpensConnection(t *testing.T) {
 	}
 }
 
+func TestEndpointEnvironmentVariableConnects(t *testing.T) {
+	os.Setenv("LOGENTRIES_ENDPOINT", "data.logentries.com")
+	le, err := Connect("")
+	os.Unsetenv("LOGENTRIES_ENDPOINT")
```

Comment:
Add a comment on why you are setting & unsetting the ENV immediately around `Connect()`